upload.py

```diff
@@ -628,8 +628,10 @@ def format_option_strings(self, option):
                  dest="save_cookies", default=True,
                  help="Do not save authentication cookies to local disk.")
 group.add_option("--oauth2", action="store_true",
-                 dest="use_oauth2", default=False,
+                 dest="use_oauth2", default=True,
```

Comment:
I think we should just delete this option as it is always true, if given on command line or not.

Reply:
> I think we should just delete this option as it is always true, if given on command line or not.

.

Reply:
f51d03f8b280c52a6ff0ef8cd5233b6082718ca5